public/app/app.ts

```diff
@@ -89,11 +104,12 @@ export class GrafanaApp {
 
   init() {
     const app = angular.module('grafana', []);
-
     addClassIfNoOverlayScrollbar();
+    setViewModeBodyClass(queryStringToJSON(getLocationService().getCurrentLocation().search).kiosk as KioskUrlValue);
```

Comment:
would be nice to build queryStringToJSON into LocationService, so we can just do locationService.getQueryParms() or something